### PR TITLE
Rwahs 117/library import

### PR DESCRIPTION
**Pull request by kehh**
Add library import script
- Export view
- import script
- sample data

Also moves the cleaning queries into `importScripts`

### CODE REVIEW

File: importScripts/sampleRecords/library.source.json [hunk 1]
```diff
@@ -0,0 +1,738 @@
...
+  "Subject": "Gee family - History; Truslove family - History.",
+  "Illustrations": "1",
+  "Maps": "0",
+  "Marks": "vi, 50p. illus., photos.",
+  "Pages": "50",
+  "Rare": "0",
+  "Restrictions": "NOT FOR LOAN",
+  "DateDue": "2002-02-14",
+  "DateOut": "2002-01-17",
+  "Borrower": "NOT FOR LOAN",
```
Comment (kehh) [inline importScripts/sampleRecords/library.source.json:26]: Not migrated - clarifying with client as to whether this field is used. See https://docs.google.com/spreadsheets/d/1tQlbe0AZos9e8mVpxY_OaJMTT3Fq-rBPWkVbIuLwvY0/edit#gid=1869772989

File: importScripts/sampleRecords/library.source.json [hunk 2]
```diff
@@ -0,0 +1,738 @@
...
+  "Value": "0"
```
Comment (kehh) [inline importScripts/sampleRecords/library.source.json:27]: Only two records have a value other than '0' and this is not included in the library UI so not migrating this field. Will clarify with the client.

File: importScripts/sampleRecords/library.destination.json [hunk 3]
```diff
@@ -0,0 +1,804 @@
...
+      "ca_objects.Notes":"2 vols.",
+      "ca_objects.PublicationType":"Books",
+      "ca_objects.DateOfPublication":"1898",
+      "ca_objects.Publisher":"Govt. Printer",
+      "ca_objects.PlaceOfPublication":"Melbourne",
+      "ca_occurrences":[
+        "Federation",
+        "Australian Federal Convention"
+      ],
+      "ca_objects.CurrentLocation":"Passage",
```
Comment (kehh) [inline importScripts/sampleRecords/library.destination.json:762]: These values have been transformed using https://docs.google.com/spreadsheets/d/1tQlbe0AZos9e8mVpxY_OaJMTT3Fq-rBPWkVbIuLwvY0/edit#gid=2068584224

File: importScripts/sampleRecords/library.source.json [hunk 4]
```diff
@@ -0,0 +1,738 @@
...
+  "WherePublished": "New Norcia",
+  "Subject": "Cottages - New Norcia - Conservation and restoration",
+  "CurrentLocation": "ANNEXE",
+  "Illustrations": "1",
+  "Maps": "0",
+  "Pages": "0",
+  "Rare": "0",
+  "Restrictions": "NOT  FOR  LOAN",
+  "DateDue": "2011-06-02",
+  "DeweyNumber": "No. 17",
```
Comment (kehh) [inline importScripts/sampleRecords/library.source.json:140]: A single record with a value, and this isn't on the new library form so this value is not migrated. Clarifying with client.

File: importScripts/sampleRecords/library.source.json [hunk 5]
```diff
@@ -0,0 +1,738 @@
...
+  "Rare": "0",
+  "Restrictions": "NOT FOR LOAN",
+  "DateDue": "2002-02-21",
+  "DateOut": "2002-01-24",
+  "Borrower": "NOT FOR LOAN",
+  "Value": "0"
+ },
+ {
+  "PublicationID": "14118",
+  "Title": "Grain pool history",
```
Comment (kehh) [inline importScripts/sampleRecords/library.source.json:202]: This record and the one below it are pretty much duplicate records. They have different PublicationIDs, but the same Accession no: in the notes field. The second record (`14119`) clobbers a lot of values from the first records as the import is set to `overwrite on idno`. I have manually duplicated the destination record in `library.destination.json` so the diff doesn't get out of whack.

File: importScripts/sampleRecords/library.source.json (hunk 1 above)
Comment (leftclickben) [inline importScripts/sampleRecords/library.source.json:26]: Did you raise an issue to clarify these unused fields?

File: importScripts/sampleRecords/library.source.json (hunk 5 above)
Comment (leftclickben) [inline importScripts/sampleRecords/library.source.json:202]: Actually I think it's better if these files go undoctored, can you regenerate this and update it?

Comment (kehh) [inline importScripts/sampleRecords/library.source.json:202]: OK I've reverted the change

**Pull request merged**